src/commands.c

```diff
@@ -882,6 +884,7 @@ cmd_t cmds[] =
 	{ "hook_smooth", 				hooksmooth, 					0, 			CF_PLAYER | CF_MATCHLESS, 											CD_HOOKSMOOTH },
 	{ "hook_fast", 					hookfast, 					0, 			CF_PLAYER | CF_MATCHLESS, 											CD_HOOKFAST },
 	{ "hook_classic", 				hookclassic, 					0, 			CF_PLAYER | CF_MATCHLESS, 											CD_HOOKCLASSIC },
+	{ "crhook", 					hookcrhook, 					0, 			CF_PLAYER | CF_MATCHLESS, 											CD_HOOKCRHOOK },
```

Comment:
I know this was the original command for this hookstyle, but it makes it more difficult to find the various options if they're not following the same naming convention.  Lets stick with the `hook_` prefix for this as well.